bin/check-stale-results.rb

```diff
@@ -60,22 +61,32 @@ def humanize(secs)
     end.compact.reverse.join(' ')
   end
 
-  def api_request(method, path)
+  def get_uri(path)
+    protocol = (settings['api'].key?('protocol') ? settings['api']['protocol'] : 'http')
+    uri = URI(protocol + '://' + settings['api']['host'] + ':' + settings['api']['port'].to_s + path)
+    uri
+  end
+
+  def api_request(path)
```

Comment:
why remote the flexibility of specifying the request type?

Reply:
reverted